packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -51,7 +52,7 @@ export const ParticipantListWithHeading = (props: {
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
 }): JSX.Element => {
   const { onFetchAvatarPersonaData, onFetchParticipantMenuItems, title, participantListProps } = props;
-  const subheadingUniqueId = useId();
```

Comment:
Can you use useId from fluent instead?

Reply:
Good suggestion! Changed to use just that